tidb-cloud/tidb-cloud-security-password-authentication.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Password Authentication
+summary: Learn how to sign up, sign in and sign out.
+---
+
+# Password Authentication
+
+Users may login in TiDB Cloud Console via Email and password. It mainly includes steps:
+
+- Step1: sign up
+- Step2: sign in and sign out
+- Step3: Password policy
+- Step4: Reset and change Password
+- Step5: Enables MFA (optional)
+
```

Comment:
Because this document is about Password Authentication, step-by-step instructions about sign-up, sign-in, and sign-out are not needed for it.